modules/popup.js

```diff
@@ -0,0 +1,73 @@
+export default class Popup {
+  static getInfos = async (id) => {
+    const movieInfo = await fetch(`https://api.tvmaze.com/shows/${id}`).then(
+      (result) => result.json(),
+    );
+    return movieInfo;
+  };
+
+  static countComments = (len) => len.length;
+
+  static display = async (movieInfo, id, comList) => {
+    const mi = await movieInfo;
+    const arr = await comList;
+    const popup = document.querySelector('.popup');
+    popup.innerHTML = `  
+      <div class="description">
+        <div class="D-description">
+          <a href="${mi.network.officialSite}"><img src="${
+      mi.image.medium
+    }" class = "image" id = "movie-img" alt=""></a>
+          <div class = "comments">
+            <div class = "form" >
+              <label for="fname">Name:</label><br>
+              <input type="text" id="fname" name="fname" placeholder = "Enter your name"><br><br>
+              <label for="comment">Comment:</label><br>
+              <textarea name="comment" id="comment" cols="20" rows="3" placeholder = "Enter comment"></textarea>
+              <input id = ${id} class= "submit" type="submit" value="Submit">
+            </div>
+          </div>
+        </div>
+        <div class = "summary-tag">
+          <h1>${mi.name}</h1>
+          ${mi.summary}
+        </div>
+      </div>
+      <ul class = "mInfo">
+        <h1 class = "remove tv_show"> TV SHOW INFO</h1>
+        <li class = "remove text_d">
+        <a class="official_site" href="${mi.network.officialSite}"> 
+        ${mi.network.name} <br>
+        </a> (${mi.premiered} - ${mi.ended})</li>
+        <li class = "remove"><b>Schedule</b>: ${mi.schedule.days[0]} at ${
+      mi.schedule.time
+    } 
+        (${mi.runtime}min)</li>
+        <li class = "remove"><b>Status</b>: ${mi.status}</li>
+        <li class = "remove"><b>Show Type:</b> ${mi.type}</li>
+        <li><b>Genres</b>: ${mi.genres}</li>
+        <li class = "remove"><b>Episodes Ordered</b> </li>
+        <li><b>language:</b>: ${mi.language}</li>
+        <li><b>Rating:</b>: ${mi.rating.average}</li>
+        <div class='show-comment'>
+          <div>
+          <h3 class = "class-heading"> All Comments (${this.countComments(
+      arr,
+    )})</h3>
+          <ul class ="D-comments">
+          </ul>
+        </div>
+      </ul>`;
```

Comment:
- [ ] Let's fix the comments count. When there are no comments, we should return 0. When a comment is added the counter should update the UI in real-time, currently, I had to refresh for me to see the count of comments.

![js](https://user-images.githubusercontent.com/53124548/197307384-7a579ba7-287c-4dce-b212-6c7c79e87f35.png)